src/assets/css/videoosd.css

```diff
@@ -6,29 +6,42 @@
     -ms-user-select: none;
 }
 
-.osdPoster img,
 .videoOsdBottom {
     bottom: 0;
     left: 0;
     right: 0;
+    position: fixed;
+    background: linear-gradient(0deg, rgba(16, 16, 16, 0.75) 0%, rgba(16, 16, 16, 0) 100%);
+    padding-top: 7.5em;
+    padding-bottom: 1.75em;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    will-change: opacity;
+    transition: opacity 0.3s ease-out;
+    color: #fff;
+    user-select: none;
+    -webkit-touch-callout: none;
 }
 
 .osdHeader {
-    -webkit-transition: opacity 0.3s ease-out;
-    -o-transition: opacity 0.3s ease-out;
     transition: opacity 0.3s ease-out;
     position: relative;
     z-index: 1;
-    background: rgba(0, 0, 0, 0.7) !important;
-    -webkit-backdrop-filter: none !important;
-    backdrop-filter: none !important;
-    color: #eee !important;
+    background: linear-gradient(180deg, rgba(16, 16, 16, 0.75) 0%, rgba(16, 16, 16, 0) 100%);
```

Comment:
The change to a gradient is making the following visible and causing the top bar on the OSD to be grey.

https://github.com/jellyfin/jellyfin-web/blob/507b4771173edaad6eb1faeedb3b167b84ca6463/src/themes/dark/theme.css#L23-L25

The style in theme.css should be removed?

Reply:
Hm, I had fixed it by overriding it with a more precise selector. Might've accidentally stashed the changes, let me check.

Reply:
There, it's fixed :)